src/session.c

```diff
@@ -509,6 +509,8 @@ static CK_RV token_session_open(P11PROV_SESSION *session, CK_FLAGS flags)
     uint64_t startime = 0;
     CK_RV ret;
 
+    P11PROV_debug("called (session=%p, flags=%x)", session, flags);
+
```

Comment:
Not sure how useful this one is, but if you think we should keep it, we need to note thatthe session is the session object pointer, not the session handle

Reply:
dropped for now